tests/async/simple.py

```diff
@@ -5,13 +5,12 @@ def __init__(self, a, b):
         self.s = "UTF-8: ❄"
 
     async def get_a_b(self):
-        s = "a is %s b is %s" % \
-            (self.a,
-            self.b)
```

Comment:
We don't want this change! The backslash is here on purpose, for coverage. I don't know how to get black to keep the backslash, so unfortunately we won't be able to run black on tests/async and tests/sync. (It's still OK to add the missing newline below.)

Reply:
I have added # fmt: off and # fmt: on to preserve the backslash.